packages/shared/src/react/hooks/useUser.ts

```diff
@@ -29,11 +29,9 @@ import { useAssertWrappedByClerkProvider, useUserContext } from '../contexts';
  * @example
  * ### Update user data
  *
- * The following example uses the `useUser()` hook to access the [`User`](https://clerk.com/docs/references/javascript/user/user) object, which calls the [`update()`](https://clerk.com/docs/references/javascript/user/user#update) method to update the current user's information.
+ * The following example uses the `useUser()` hook to access the [`User`](https://clerk.com/docs/references/javascript/user) object, which calls the [`update()`](https://clerk.com/docs/references/javascript/user#update) method to update the current user's information.
  *
  * ```tsx {{ filename: 'src/Home.tsx' }}
- * import { useUser } from '@clerk/clerk-react'
```

Comment:
The removal is on purpose, our docs will inject this at some point depending on which SDK you choose